.travis.yml

```diff
@@ -104,15 +104,6 @@ install:
   - echo "[System]" >> $SITE_CFG
   - echo "udunits2_path = $PREFIX/lib/libudunits2.so" >> $SITE_CFG
 
-  # The coding standards tests expect all the standard names and PyKE
-  # modules to be present.
-  - if [[ $TEST_TARGET == 'coding' ]]; then
-      python setup.py std_names;
-      PYTHONPATH=lib python setup.py pyke_rules;
-    fi
-
-# iris
-  - python setup.py --quiet build
   - python setup.py --quiet install
```

Comment:
The `test_pep8_conformance` is failing as (I think) the rule are not being compiled and installed.

On `install` it is compiling the rules but (I think) they are relative to the root i.e. in the same directory as the `setup.py`, and thus not being installed, hence why on installation there are no compiled rules for the kb.

The the problem in within the `setup.py` refactor of the `compile_pyke_rules`